doc/api/icu.md

```diff
@@ -0,0 +1,76 @@
+# icu
+
+    Stability: 1 - Experimental
+
+The `icu` module provides an interface to Unicode and Internationalization
+functionality provided by the ICU4C library.
+
+```js
+const icu = require('icu');
+```
+
+## icu.transcode(buf, from_enc, to_enc)
+
+* `buf` {Buffer} A `Buffer` instance
+* `from_enc` {string} The current encoding
+* `to_enc` {string} The target encoding
+
+Re-encodes the given `Buffer` from one character encoding to another. Returns
+a new `Buffer` instance.
```

Comment:
would it be possible to allow an optional `buffer` argument so another one doesn't need to be created? not for this PR, but would be a nice addition in the future.